baked_in.go

```diff
@@ -1404,6 +1407,74 @@ func isFile(fl FieldLevel) bool {
 	panic(fmt.Sprintf("Bad field type %T", field.Interface()))
 }
 
+func isImage(fl FieldLevel) bool {
+	mimetypes := map[string]bool{
+		"image/bmp": true,
+		"image/cis-cod": true,
+		"image/gif": true,
+		"image/ief": true,
+		"image/jpeg": true, 
+		"image/jp2": true, 
+		"image/jpx": true, 
+		"image/jpm": true, 
+		"image/pipeg": true,
+		"image/png": true, 
+		"image/svg+xml": true,
+		"image/tiff": true,
+		"image/webp": true,
+		"image/x-cmu-raster": true,
+		"image/x-cmx": true,
+		"image/x-icon": true,
+		"image/x-portable-anymap": true,
+		"image/x-portable-bitmap": true,
+		"image/x-portable-graymap": true,
+		"image/x-portable-pixmap": true,
+		"image/x-rgb": true,
+		"image/x-xbitmap": true,
+		"image/x-xpixmap": true,
+		"image/x-xwindowdump": true,
+	}
+	field := fl.Field()
+
+	switch field.Kind() {
+	case reflect.String:
+		filePath := field.String()
+		fileInfo, err := os.Stat(filePath)
+
+		if err != nil {
+			return false
+		}
+
+		if fileInfo.IsDir() {
+			return false
+		}
+
+		file, err := os.OpenFile(filePath, os.O_RDWR, 0644)
```

Comment:
No need to have write permissions that I can see and `os.Open(...)` uses `O_RDONLY`
```suggestion
		file, err := os.Open(filePath)
```